source/specifications.rst

```diff
@@ -94,17 +94,12 @@ be ``UTF-8``.
 
 Other parameters might be specific to the chosen subtype. For example, for the
 ``markdown`` subtype, there is a ``variant`` parameter that allows specifying
-the variant of Markdown in use, such as:
+the variant of Markdown in use. Currently, the only value that is recognized
```

Comment:
Is variant mandatory? If it is can we call that out, and if not can we mention that if variant is missing then it is assumed to be CommonMark? (In case we add more in the future).